app/modules/sightings/models.py

```diff
@@ -257,7 +292,139 @@ def rectify_edm_encounters(self, edm_encs_json, user=None):
             )
             self.add_encounter(encounter)
 
-    def ia_pipeline(self):
-        # TODO DEX-296 Trigger the sighting to start Identification if annotations present or to go to
-        # un reviewed if not
-        pass
+    def get_matching_set_data(self, matching_set_data):
+        annots = []
+
+        # Must match the options validated in the metadata.py
+        if matching_set_data == 'mine':
+            data_owner = self.single_encounter_owner()
+            assert data_owner
+            annots = data_owner.get_my_annotations()
+        elif matching_set_data == 'extended':
+            data_owner = self.single_encounter_owner()
+            assert data_owner
+            annots = data_owner.get_all_annotations()
+        elif matching_set_data == 'all':
+            from app.modules.annotations.models import Annotation
```

Comment:
Perhaps a discussion for Friday: do we want to handle matching set queries separate from the other searches? It would be pretty cool to leverage elasticsearch for searching across encounters to return annotations for a matching set. 

Returning all annotations can potentially be an enormous amount of data, we should probably limit that list by anything we have available.

Reply:
I understand the elasticsearch functionality isn't available yet so there might just need to be a spopgap.

Reply:
We did in Wildbook have all the matching set code on the annotation as a classmethod and private methods, so you could ask each annotation what it's matching set was. We should consider that probably.

Reply:
This is utterly noddy at the moment with the intention that it could be enhanced in the future.
That's sort of why it's in a separate method.